es/introduction.md

```diff
@@ -94,15 +94,15 @@ Para utilizar el código público existente, hay que especificar en el presupues
 
 ## Cumplimiento de normas o proceso de certificación
 
-La Foundation for Public Code se asegura de que las codebase bajo su tutela o stewardship (y no *in incubation* o *in the attic* ) cumplan con el Estándar para el Código Público. Esto deja claro a los usuarios y colaboradores potenciales que la codebase es de alta calidad, y que las actualizaciones también lo serán.
+La Foundation for Public Code se asegura de que las codebase bajo su tutela o dirección (y no *in incubation* o *in the attic* ) cumplan con el Estándar para el Código Público. Esto deja claro a los usuarios y colaboradores potenciales que la codebase es de alta calidad, y que las actualizaciones también lo serán.
```

Comment:
We don't identify stewardship as direction, so we'd keep the original in this suggestion